middleware/login_queries.py

```diff
@@ -0,0 +1,60 @@
+import jwt
+import os
+import datetime
+
+
+def login_results(cursor, email):
+    cursor.execute(
+        f"select id, password_digest, api_key from users where email = '{email}'"
+    )
+    results = cursor.fetchall()
+    if len(results) > 0:
+        user_data = {
+            "id": results[0][0],
+            "password_digest": results[0][1],
+            "api_key": results[0][2],
+        }
+        return user_data
+    else:
+        return {"error": "no match"}
+
+
+def is_admin(cursor, email):
+    cursor.execute(f"select role from users where email = '{email}'")
+    results = cursor.fetchall()
+    if len(results) > 0:
+        role = results[0][0]
+        if role == "admin":
+            return True
+        return False
+
+    else:
+        return {"error": "no match"}
+
+
+def create_session_token(cursor, id, email):
+    expiration = datetime.datetime.utcnow() + datetime.timedelta(days=0, seconds=300)
+    payload = {
+        "exp": expiration,
+        "iat": datetime.datetime.utcnow(),
+        "sub": id,
+    }
+    session_token = jwt.encode(payload, os.getenv("secret_key"), algorithm="HS256")
```

Comment:
@mbodeantor looks like we're looking for `secret_key` rather than `SECRET_KEY`. When I change this line, the client is fine locally with the uppercased value. 

```suggestion
    session_token = jwt.encode(payload, os.getenv("SECRET_KEY"), algorithm="HS256")
```

Do you prefer this var to be exported in snake case rather than screaming snake?

Reply:
Ah no is should be uppercase, forgot to change it back after trying to get it to work in GH.

Reply:
Alright, I committed it. 🙏